client/src/index.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/app.jsx';
+
+
+ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
@stellaliang21 Good job starting with a simple component to get everything working!